.npmignore

```diff
@@ -0,0 +1,23 @@
+.idea/
+api-doc/
+apps/
+design-notes/
+examples/
+node_modules/
+performance/
+src/
+test/
+test-results/
+tools/
+.gitignore
+.npmignore
+.travis.yml
+config.json
+Gruntfile.js
+HowToBuildWebWW.md
+HowToCreateAndRunUnitTests.md
+karma.conf.js
+layout.tmpl
+package-lock.json
+package.json
```

Comment:
Thanks for the reference. I'll remove them. Do you think I should separate everything in .npmignore with newlines as in .gitignore?